docs/test-framework-dev-guide.md

```diff
@@ -136,28 +136,70 @@ when running them manually, such as `ELASTICSEARCH_HOST`, `ELASTICSEARCH_USERNAM
 
 ### Debugging tests
 
-#### Manually debugging tests on VMs
-Many of the integration tests will install the Elastic-Agent and/or
-require root to run, which makes it hard to just run them on our work
-machines, the best way to circumvent that is to debug the tests
-directly on a VM. `mage integration:DeployDebugTools` will show a menu
-to select a VM and then install the common debugging tools: Delve,
-Mage and Docker. It will also create the `~/elastic-agent` folder
-containing the Git repository (required to package from within the VM)
+#### Connecting to VMs
+All VMs (including Windows) support connections via SSH, the framework
+generates and stores the necessary SSH keys to access the VMs, the
+easiest way to connect to them is using the SSH command returned by
+`mage integration:SSHVM`. It will list the VMs and ask to select
```

Comment:
I'm following the onboarding documentation and noticed that `integration:SSHVM` is not a valid mage target. The correct one seems to be `integration:ssh`. Could you please update it? Thanks.

Reply:
Thank you for spotting that! I fixed it on [a4a2f16](https://github.com/elastic/elastic-agent/pull/5461/commits/a4a2f16739af41196072f3c895ea292ed097a95f)